src/main/java/edu/hawaii/its/groupings/configuration/OotbStaticUserAuthenticationFilter.java

```diff
@@ -0,0 +1,44 @@
+package edu.hawaii.its.groupings.configuration;
+
+import java.io.IOException;
+import java.util.List;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.GenericFilterBean;
+
+public class OotbStaticUserAuthenticationFilter extends GenericFilterBean {
+    private final String serverType;
+    private final String name;
+    private final String username;
+    private final List<String> authorities;
+
+    public OotbStaticUserAuthenticationFilter(String serverType, String name, String username,
+            List<String> authorities) {
+        this.serverType = serverType;
+        this.name = name;
+        this.username = username;
+        this.authorities = authorities;
+    }
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain filterChain)
+            throws IOException, ServletException {
+        if ("OOTB".equals(serverType) && SecurityContextHolder.getContext().getAuthentication() == null) {
```

Comment:
Could you invert the the `.equals` from `"OOTB".equals(serverType)` to `serverType.equals("OOTB")`? Thanks!